### PR TITLE
[NN Models] Bash scripts for getting models

### PR DESCRIPTION
**Pull request by hjbc0921**
This commit provides bash scripts for getting
NN models used in nnstreamer-example.

Signed-off-by: hjbc0921 <hjbc0921@gmail.com>

### CODE REVIEW

File: bash_script/example_models/get-model-object-detection-tf.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
+mkdir tf_model
```
Comment (myungjoo) [inline bash_script/example_models/get-model-object-detection-tf.sh]: This returns error if ```get-model-object-detection-tflite.sh``` is already executed. Add -p option

File: bash_script/example_object_detection_tensorflow/gst-launch-object-detection-tf.sh [hunk 2]
```diff
@@ -2,11 +2,11 @@
 gst-launch-1.0 \
 	v4l2src name=cam_src ! videoscale ! videoconvert ! video/x-raw,width=640,height=480,format=RGB,framerate=30/1 ! tee name=t \
     t. ! queue leaky=2 max-size-buffers=2 ! videoscale ! tensor_converter ! \
-		tensor_filter framework=tensorflow model=ssdlite_mobilenet_v2.pb \
+		tensor_filter framework=tensorflow model=tf_model/ssdlite_mobilenet_v2.pb \
```
Comment (myungjoo) [inline bash_script/example_object_detection_tensorflow/gst-launch-object-detection-tf.sh]: Is this path still correct with the updated bash locations and model paths?

File: bash_script/meson.build [hunk 3]
```diff
@@ -5,3 +5,5 @@ endif
 if have_tensorflow_lite
   subdir('example_object_detection_tensorflow_lite')
 endif
+
+subdir('example_models')
```
Comment (myungjoo) [inline bash_script/meson.build]: We don't have meson.build in example_models directory. It appears that you didn't execute ```git add ....../meson.build```.

File: bash_script/example_object_detection_tensorflow/gst-launch-object-detection-tf.sh (hunk 2 above)
Comment (hjbc0921) [reply bash_script/example_object_detection_tensorflow/gst-launch-object-detection-tf.sh]: All bash scripts will be in $NNST_ROOT/bin folder, and in that case this model paths are correct. Also I confirmed that this runs with no error.

**Pull request merged**